modules/ting/src/Search/TingSearchRequest.php

```diff
@@ -233,6 +233,44 @@ public function setTermsPrFacet($terms_per_facet) {
     return $this;
   }
 
+  /**
+   * The facets used for this search request.
+   *
+   * @see TingSearchRequest::setFacets() for more information.
+   *
+   * @return string[]
+   *   The facets.
+   */
+  public function getFacets() {
+    return $this->facets;
+  }
+
+  /**
+   * Sets the facets used for this search request.
```

Comment:
This is still vague to me: `the facets used for this search request`. What does *used for this search request* mean? Should it affect the results returned by the provider?

As I see it the key purpose of this function is to set which facets we want returned with the search result.